test/IdentityManager.js

```
@@ -392,4 +392,153 @@ contract('IdentityManager', (accounts) => {
       })
     })
   })
+  describe('migration', () => {
+    let newIdenManager
+    beforeEach(done => {
+      identityManager.CreateIdentity(user1, recoveryKey, {from: nobody}).then(tx => {
+        const log = tx.logs[0]
+        assert.equal(log.event, 'IdentityCreated', 'wrong event')
+        proxy = Proxy.at(log.args.identity)
+        return IdentityManager.new(userTimeLock, adminTimeLock, adminRate)
+      }).then((instance) => {
+        newIdenManager = instance
+        //user2 is now a younger owner, while user1 is an olderowner
+        return identityManager.addOwner(proxy.address, user2)
+      }).then(tx => {
+        let log = tx.logs[0]
+        assert.equal(log.event, 'OwnerAdded', 'wrong event')
+        assert.equal(log.args.identity, proxy.address, 'wrong proxy')
+        assert.equal(log.args.owner, user2, 'wrong owner added')
+        assert.equal(log.args.instigator, user1, 'wrong initiator')
+        done()
+      }).catch(e => {
+        console.log(e)
+        done()
```

This is the place where it will swallow the errors from the asserts.